script.tag-generator/resources/lib/trakt/errors.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
Isn't this already available from script.module.trakt? @razzeee

Reply:
I think this is Pytrakt, I'm using Trakt.py